demo-app/angular.json

```diff
@@ -64,24 +64,24 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "demo-app:build",
             "host": "localhost",
-            "port": 4200
+            "port": 4200,
+            "buildTarget": "demo-app:build"
```

Comment:
Just a formatting issue. Please do not change the order of these settings if possible. We do not want to highligh the `port: 4200` line as change.

Reply:
Is renaming from "browserTarget" to "buildTarget" required? It's not a problem obviously, just interesting

Reply:
Thank you for review. "browserTarget" is now deprecated https://angular.io/cli/serve

Reply:
It was automatically replaced when I ran "ng update"